.travis.yml

```diff
@@ -3,14 +3,17 @@
 #    - CONDA_PREFIX=$HOME/miniconda
 #    - MINICONDA_URL_BASE="https://repo.continuum.io/miniconda/Miniconda3-latest"
 
+
 sudo: false
 
 language: python
 
 matrix:
   include:
     - os: linux
-      python: 2.7
+      python: 2.7.10
```

Comment:
Having 2 different 2.7 versions may be overkill, but I had a lot of builds failing on just one of these...